scipy/misc/pilutil.py

```diff
@@ -28,6 +27,14 @@
 __all__ = ['fromimage', 'toimage', 'imsave', 'imread', 'bytescale',
            'imrotate', 'imresize', 'imshow', 'imfilter']
 
+from ndimage import imresize, imfilter, imrotate
+imresize = numpy.deprecate(imresize,
+                           message='misc.imresize is deprecated in Scipy 1.0')
```

Comment:
gh-6242 had some discussion on whether these functions shouldn't just be deprecated unconditionally (https://github.com/scipy/scipy/issues/6242#issuecomment-225426614). There's something to say for that. Good to get a decision on that first.